api/route/Article/article.summary.ts

```diff
@@ -4,16 +4,22 @@ import { getClient } from '../../database';
 import { Route, Error, Success } from '../router';
 import { Article } from '@shared';
 
-const ARTICLES_PER_CATEGORY = 4;
+const ARTICLES_PER_CATEGORY = 3;
+
+// query shows three items from each category ranking from most clicks to least, 
+// if no clicks registered for articles, orders by publication date
 const groupCategoriesInThrees = `
-  select id, name, author, publication_date, category, source_url, cover_url
-  from (
-    select ar.id, a.name, ar.author, a.body, ar.category, ar.source_url, ar.cover_url, ar.retrieved_date, ar.publication_date, 
-      row_number() over (PARTITION BY ar.category order by ar.publication_date) index_in_category
+  select * from (
```

Comment:
can we swap `*` out with just the fields we need. Even if we need all of them its still faster to list them explicitly over using `*`.